wiki/Article_Style_Guide/en.md

```diff
@@ -101,26 +170,46 @@ For example use:
 - `skilful` instead of `skillful`
 - `analyse` instead of `analyze`
 
-### Capitalisation
-
-#### osu!
+### osu!
 
 The name of the game `osu!` is **never capitalised**, even if it is the first word in the sentence.
-Any words following `osu!`, if not divided by a blank space, should not be capitalised (i.e _see game mode spellings below_).
-If they are divided by a blank space, they must be capitalised if they are proper nouns (i.e. osu! Tournaments).
+If `osu!` is the last word in the sentence, add a period (or the appropriate punctuation marks) immediately following `osu!`.
+For example:
+
+- `The welcome.wav file says, "Welcome to osu!."`
+
+Any words following `osu!`, if not divided by a blank space, should not be capitalised.
+If they are divided by a blank space, they must be capitalised if they are proper nouns.
+For example:
+
+- `osu! Tournaments`
+- `osu! Alumni`
 
 If you have `osu!` then a word immediately following it, make sure that the word is spelt with lowercase letters (like the game modes).
 Other examples may include:
 
-- `osu!wiki`
-- `osu!alumni`
+- `osu!mania`
 - `osu!direct`
+- `osu!wiki`
+
+When refering to `osu!` (the game/framework itself, not the game mode), it should be in _italics_ unless it is included in the name of game modes or other services related to the game.
+For example:
+
+- `The language that _osu!_ is written in is C# using the .NET Framework.`
+
+### Serial comma
+
+When listing items of 3 or more in a sentence, use the serial comma.
+
+- The game modes of _osu!_: osu!standard, osu!taiko, osu!catch**,** and osu!mania are fun to play with others.
```

Comment:
~~❓ I'm not really sure on what you mean about those `**`?~~ oh wait, are you saying that the missing space between `catch` and `**` is causing it to not be styled or something?